media-libs/vidstab/vidstab-9999.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit git-r3 cmake-utils
+
+DESCRIPTION="Video stabilization library"
+HOMEPAGE="http://public.hronopik.de/vid.stab/"
+EGIT_REPO_URI="https://github.com/georgmartius/vid.stab.git"
+EGIT_BRANCH="master"
```

Comment:
Since master is the default branch, please don't specify it explicitly.

Reply:
Ok, will fix shortly.